troubleshooting.md

```diff
@@ -916,6 +916,209 @@ After deleting a VM on macOS, the initialization of subsequent VMs fails.
 
 After deleting a client VM on macOS via `podman machine stop` && `podman machine rm`, attempting to `podman machine init` a new client VM leads to an error with the 127.0.0.1:7777 port already bound.
 
-### Solution
+#### Solution
 
 You will need to remove the hanging gv-proxy process bound to the port in question. For example, if the port mentioned in the error message is 127.0.0.1:7777, you can use the command `kill -9 $(lsof -i:7777)` in order to identify and remove the hanging process which prevents you from starting a new VM on that default port.
+
+### 32) Container creates a file that is not owned by the user's regular UID
+
+#### Symptom
```

Comment:
After running a container with rootless Podman, the non-root user sees a numerical UID and GID instead of a username and groupname.

#### Symptom

When listing file permissions with `ls -l` on the host in a directory that was passed as `--volume /some/dir` to `podman run`, the UID and GID are displayed rather than the corresponding username and groupname.  The UID and GID numbers displayed are from the user's UID and GID ranges on the host system.